src/app/pages/HomePage/index.tsx

```diff
@@ -180,6 +181,7 @@ export const HomePage: FC = () => {
               />
             </Box>
           </ThemeByNetwork>
+          <CookieConsent />
```

Comment:
Doesn't appear in subpages http://localhost:1234/mainnet/sapphire
![localhost_1234_mainnet_emerald_token_0x903d32d7307b4b3FC4bc9a510CA658C91A346A67](https://github.com/oasisprotocol/explorer/assets/3758846/7329a2cf-8679-4352-b8fd-13bf745811d1)